composer.json

```diff
@@ -15,19 +15,21 @@
             "homepage": "http://github.com/KnpLabs/KnpSnappyBundle/contributors"
         }
     ],
-    "minimum-stability": "dev",
```

Comment:
We should add it back but, unfortunately, Snappy dev-master has some breaking changes (my bad, I'm gonna fix it) and, in order to make tests pass now we have to remove it (some of the tests don't use `--prefer-stable` composer flag, that's why they fail).